src/components/Products/Products.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import {ProductsWrapper, CardContainer,Card, CardText } from './Products.styled';
+
+export default function Products({products}) {
+    return (
+            <ProductsWrapper>
+                <CardContainer>
+                {products.map((product, index) => (
+                    <Card 
+                        key={index}  
```

Comment:
this code should concat string with the index to avoid react js warning by the key (just index)